auditbeat/auditbeat.reference.yml

```diff
@@ -723,7 +723,7 @@ output.elasticsearch:
 #setup.dashboards.index:
 
 # Force loading of dashboards using the Kibana API without querying Elasticsearch for the version
```

Comment:
WDYT about rewording this to not include "force"? @kvch 

```
# Always use the Kibana API for loading the dashboards instead of autodetecting
# how to install the dashboards by first querying Elasticsearch.
```

Reply:
Yes, much better. The name of the option and its functionality changed slightly since I added those line. :)